control-plane/build-support/functions/10-util.sh

```diff
@@ -108,70 +105,34 @@ function parse_version {
 	#   If the GOTAGS environment variable is present then it is used to determine which
 	#   version file to use for parsing.
 
-	local vfile="${1}/version/version.go"
+	local vfile="${1:-.}/version/VERSION"
 
 	# ensure the version file exists
 	if ! test -f "${vfile}"; then
 		err "Error - File not found: ${vfile}"
 		return 1
 	fi
 
-	local include_release="$2"
-	local use_git_env="$3"
-	local omit_version="$4"
-
```

Comment:
These were always `true`, `false`, and unset, respectively (2 uses of `parse_version`, both in `40-publish.sh`)